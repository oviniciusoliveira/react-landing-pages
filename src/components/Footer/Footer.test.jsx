import { renderTheme } from '../../test/render-theme';
import { Footer } from '.';
import { screen } from '@testing-library/dom';

describe('<Footer />', () => {
  test('should render with html text', () => {
    renderTheme(<Footer footerHtml="<h1>Footer Text <h1/>" />);
    expect(screen.getByRole('heading', { name: 'Footer Text' })).toBeInTheDocument();
  });

  test('should match snapshot', () => {
    const { container } = renderTheme(<Footer footerHtml="<h1>Footer Text <h1/>" />);
    expect(container).toMatchSnapshot();
  });
});
