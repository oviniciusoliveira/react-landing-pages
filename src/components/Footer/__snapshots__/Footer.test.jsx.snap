// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
.c3 {
  font-size: 2.4rem;
}

.c2 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

.c0 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c0 a {
  color: inherit;
  font-size: 1.6rem;
}

.c0 .c1 {
  padding-top: 0;
  padding-bottom: 0;
}

<div>
  <footer
    class="c0"
  >
    <div
      class="c1 c2"
    >
      <div
        class="c3"
      >
        <h1>
          Footer Text 
        </h1>
        <h1 />
      </div>
    </div>
  </footer>
</div>
`;
